examples/NimBLE_iBeacon/NimBLE_iBeacon.ino

```diff
@@ -0,0 +1,46 @@
+/**
+ *  iBeacon example
+ *
+ *  This example demonstrates how to publish an Apple-compatible iBeacon
+ *
+ *  Created: on May 26 2025
+ *      Author: lazd
+ */
+
+#include <Arduino.h>
+#include <NimBLEDevice.h>
+#include <NimBLEBeacon.h>
+
+// According to Apple, it's important to have a 100ms advertising time
+#define BEACON_ADVERTISING_TIME	160   // 100ms
+
+// Hey, you! Replace this with your own unique UUID with something like https://www.uuidgenerator.net/
+const char* iBeaconUUID = "26D0814C-F81C-4B2D-AC57-032E2AFF8642";
+
+void setup() {
+	NimBLEDevice::init("NimBLEiBeacon");
+
+	// Create beacon object
+	NimBLEBeacon beacon;
+	beacon.setManufacturerId(0x4C00); // fake Apple 0x004C LSB (ENDIAN_CHANGE_U16!)
```

Comment:
The fact that you have to reverse endianness here can be confusing, but it's the same as Bluedroid.

Reply:
You don't even need to set this, the default construction will do it for you 😄 .